### PR TITLE
docs(common): razor components scenario migration tip

### PR DESCRIPTION
**Pull request by marin-bratanov**

### CODE REVIEW

**Comment by marin-bratanov** [comment]
Bump @telerik/blazor-support

File: common/razor-components/readme.md [hunk 1]
```diff
@@ -8,3 +8,15 @@ Here are the key points (comments in the code offer more details):
...
 * **At the time of writing, there is a problem in the framework with passing parameters to components which results in errors**. For the time being, components need to get their own data. You should monitor the framework for fixes and updates in this regard. This may be related to [https://github.com/aspnet/AspNetCore/issues/14966](https://github.com/aspnet/AspNetCore/issues/14966) and [https://github.com/aspnet/AspNetCore/issues/14474](https://github.com/aspnet/AspNetCore/issues/14474) even though both issues are closed and expected to be fixed in .NET Core 3.1 Preview 2, which this sample uses.
 * Keep in mind the following Telerik-specific considerations: [Telerik UI for Blazor in an ASP.NET Web Application](https://docs.telerik.com/blazor-ui/knowledge-base/blazor-in-asp-net).
+
+
+
+## Notes
+
+Note the usage of the `TelerikRootComponent` - it is required for popups (such as windows, dropdowns, alerts, notifications) and its placing is important for their correct positioning (read more about this in the <a href="https://docs.telerik.com/blazor-ui/troubleshooting/general-issues#wrong-popup-position" target="_blank">Wrong Popup Position</a> section).
```
Comment (dimodi) [inline common/razor-components/readme.md:16]:
```suggestion
Note the usage of the `TelerikRootComponent` - it is required for popups (such as windows, dialogs, dropdowns, alerts, notifications). Its placing is important for correct popup positioning (read more about this in the <a href="https://docs.telerik.com/blazor-ui/troubleshooting/general-issues#wrong-popup-position" target="_blank">Wrong Popup Position</a> section).
```

File: common/razor-components/readme.md [hunk 2]
```diff
@@ -8,3 +8,15 @@ Here are the key points (comments in the code offer more details):
...
 * Keep in mind the following Telerik-specific considerations: [Telerik UI for Blazor in an ASP.NET Web Application](https://docs.telerik.com/blazor-ui/knowledge-base/blazor-in-asp-net).
+
+
+
+## Notes
+
+Note the usage of the `TelerikRootComponent` - it is required for popups (such as windows, dialogs, dropdowns, alerts, notifications). Its placing is important for correct popup positioning (read more about this in the <a href="https://docs.telerik.com/blazor-ui/troubleshooting/general-issues#wrong-popup-position" target="_blank">Wrong Popup Position</a> section).
+
+With the razor components scenarios, it is impossible to have one such component at the root of the app, like you can have with a "standard" Blazor application. Each Razor Component in this scenario is its own standalone and isolated Blazor application and they can't share rendering and components.
```
Comment (dimodi) [inline common/razor-components/readme.md:18]:
```suggestion
With the razor components scenarios, it is impossible to have one `TelerikRootComponent` at the root of the app, like you can have with a "standard" Blazor application. Each Razor Component in this scenario is its own standalone and isolated Blazor application and they can't share rendering and components.
```

File: common/razor-components/readme.md [hunk 3]
```diff
@@ -8,3 +8,15 @@ Here are the key points (comments in the code offer more details):
 * You need to include the Blazor assets and start up the Blazor hub yourself (you can find them in the layout).
 * **At the time of writing, there is a problem in the framework with passing parameters to components which results in errors**. For the time being, components need to get their own data. You should monitor the framework for fixes and updates in this regard. This may be related to [https://github.com/aspnet/AspNetCore/issues/14966](https://github.com/aspnet/AspNetCore/issues/14966) and [https://github.com/aspnet/AspNetCore/issues/14474](https://github.com/aspnet/AspNetCore/issues/14474) even though both issues are closed and expected to be fixed in .NET Core 3.1 Preview 2, which this sample uses.
 * Keep in mind the following Telerik-specific considerations: [Telerik UI for Blazor in an ASP.NET Web Application](https://docs.telerik.com/blazor-ui/knowledge-base/blazor-in-asp-net).
+
+
```
Comment (dimodi) [inline common/razor-components/readme.md:12]:
```suggestion
```

**Review by dimodi — APPROVED** [review]

**Pull request merged**